validator/client/service.go

```diff
@@ -186,6 +190,7 @@ func (v *ValidatorService) Start() {
 		emitAccountMetrics:             v.emitAccountMetrics,
 		startBalances:                  make(map[[fieldparams.BLSPubkeyLength]byte]uint64),
 		prevBalance:                    make(map[[fieldparams.BLSPubkeyLength]byte]uint64),
+		pubkeyHexToValidatorIndex:      make(map[string]types.ValidatorIndex),
```

Comment:
Would it be better if you change it to `make(map[[48]byte]types.ValidatorIndex)`?

Seems like we could avoid some unnecessary conversations

Reply:
i don't think maps can use bytes as the key 🤔